tools/install-unprivileged.sh

```diff
@@ -156,7 +196,7 @@ LASTPKGS=""
 latesturl()
 {
 	typeset URL="$1"
-	if [ "$3" = true ]; then
+	if [ "$3" == true ]; then
```

Comment:
If you make this kind of change please make it consistently.  I see for example below `[ "$4" = true ]`.